mailet/standard/src/main/java/org/apache/james/transport/mailets/LogMessage.java

```diff
@@ -73,9 +136,15 @@ public void init() {
             headers = getInitParameter("headers", true);
             body = getInitParameter("body", true);
             bodyMax = (getInitParameter("maxBody") == null) ? 0 : Integer.parseInt(getInitParameter("maxBody"));
+            if(bodyMax < 0){
+                logger.error("Parsed a negative maxBody value and caused initialization failure");
+                throw new IllegalArgumentException("The value of maxBody cannot be negative.");
+            }
             comment = getInitParameter("comment");
-        } catch (Exception e) {
-            logger.error("Caught exception while initializing LogMessage", e);
+        } catch (NumberFormatException e){
+            logger.error("Caught exception while parsing maxBody value", e);
```

Comment:
I would argue that we don't want to start with an invalid configuration. Maybe catching these exceptions is not needed?